run-make-check.sh

```diff
@@ -66,7 +66,7 @@ function run() {
 	$DRY_RUN ./install-deps.sh || return 1
     fi
     $DRY_RUN ./autogen.sh || return 1
-    $DRY_RUN ./configure "$@"  --with-librocksdb-static --disable-static --with-radosgw --with-debug --without-lttng \
+    $DRY_RUN ./configure "$@"  --with-librocksdb-static --disable-static --with-radosgw --with-debug \
```

Comment:
OMG, it's just a stupid "git commit -a " after a testing with lttng. Really sorry for that, I should check it carefully after pushing to github. thanx @dillaman